allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -8,7 +8,7 @@ import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
     <div class="widget__handle">
-        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
+       <span class="draggable-icon"></span>
```

Comment:
1. Block widget have element - handle. Which have icon block draggable. So we can change area of element "handle" without touching icon